internal/config/sim_control.go

```diff
@@ -113,8 +113,71 @@ func (sc *simControl) SetRobotDimentions() {
 	fmt.Println("Not yet implemented")
 }
 
-func (sc *simControl) RobotStartPositionConfig1(numberOfRobots int) {
-	fmt.Println("Not yet implemented")
+func (sc *simControl) RobotStartPositionConfig1(yellowCoords [][2]float32, blueCoords [][2]float32) {
+	orientation := float32(0.0) // Approx. 45 degrees in radians
+	vx := float32(0.0)          // Velocity towards x-axis
+	vy := float32(0.0)          // Velocity towards y-axis
+	vAngular := float32(0.0)    // Angular velocity
+
+	robotList := []*simulation.TeleportRobot{}
+
+	for i, pos := range yellowCoords {
```

Comment:
A great addition to the project! I just have a few comments to avoid potential hassle in the future. 

Feel free to ignore this comment

Both for loops do exactly the same thing. It would be great to extract that logic into a function or make an outer loop over Coords arrays so we can reduce the number of lines we need to maintain and keep it a bit more DRY